registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/MessageBusAddress.java

```diff
@@ -211,8 +211,8 @@ public void setAddress(String address) {
 	public static final MessageBusAddress PACKET_CLASSIFIER_BUS_OUT = new MessageBusAddress("packet-classifier-bus-out");
 	
 	/** The Constant WORKFLOW_EVENTUPDATE_BUS_IN. */
-	public static final MessageBusAddress WORKFLOW_EVENTUPDATE_BUS_IN = new MessageBusAddress(
-			"workflow-eventupdate-bus-in");
+	public static final MessageBusAddress WORKFLOW_EVENT_UPDATE_ADDRESS = new MessageBusAddress(
+			"workflow-event-update-address");

```

Comment:
In the actual address lets not have the 'address', lets it be "workflow-event-update"